code/datums/helper_datums/pconstruction_datum.dm

```diff
@@ -0,0 +1,260 @@
+#define FORWARD -1
+#define BACKWARD 1
+
+/*
+This only exists so I don't break mecha construction
+because the system mecha uses sucks horribly
+
+so i'm using paradise's superior construction datum
+
+which looks 100x better in practice and is just generally better
+*/
+
+/datum/pconstruction
```

Comment:
Compare mecha construction code: https://github.com/MoreRobustThanYou/tgstation/blob/spacepod/code/game/mecha/mecha_construction_paths.dm

with spacepod construction code: https://github.com/MoreRobustThanYou/tgstation/blob/spacepod/code/modules/spacepods/construction.dm

Spacepod is much cleaner with pconstruction, no constant switch checks, etc